near-sdk/Cargo.toml

```diff
@@ -32,6 +32,12 @@ wee_alloc = { version = "0.4.5", default-features = false, optional = true }
 # Used for caching, might be worth porting only functionality needed.
 once_cell = { version = "1.8", optional = true, default-features = false }
 
+[dependencies.near-abi]
+git = "https://github.com/near/near-abi-rs"
+branch = "miraclx/init"
+features = ["__chunked-entries"]
+optional = true
```

Comment:
Everything seems fine, just want to make sure this doesn't come in until it points at a release (so blocked on https://github.com/near/near-abi-rs/pull/1)